fe/fe-core/src/main/java/com/starrocks/analysis/ShowComputeNodesStmt.java

```diff
@@ -0,0 +1,45 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+import com.starrocks.catalog.Column;
+import com.starrocks.catalog.ScalarType;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.common.ErrorCode;
+import com.starrocks.common.ErrorReport;
+import com.starrocks.common.proc.ComputeNodeProcDir;
+import com.starrocks.mysql.privilege.PrivPredicate;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.qe.ShowResultSetMetaData;
+import com.starrocks.server.GlobalStateMgr;
+import com.starrocks.sql.ast.AstVisitor;
+
+public class ShowComputeNodesStmt extends ShowStmt {
```

Comment:
new statement should be put 'fe/fe-core/src/main/java/com/starrocks/sql/ast'

Reply:
ok